package.json

```diff
@@ -59,30 +60,5 @@
     "*.{ts,tsx}": [
       "eslint --fix"
     ]
-  },
-  "workspaces": [
```

Comment:
what happened here @lachlancollins ? Our workspaces and overrides are gone...

Reply:
The only workspace config needed is in `pnpm-workspace.yaml`. pnpm also handles all overrides for local packages via `linkWorkspacePackages: true` and `preferWorkspacePackages: true`

Reply:
Hmmm that shouldn't be an issue, the examples should still load workspace versions if configured correctly - let me check!

Reply:
I tested it some more and apparently the behavior is the workspace version is pulled in when it matches. There's perhaps other cases where it wasn't getting pulled into which I ran into before but in any case, yeah I think we're fine.